docs/sources/tempo/setup/operator/_index.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Deploy Tempo with Tempo Operator
+menuTitle: Deploy with operator
+description: Learn how to deploy Tempo with Tempo Operator
+weight:
```

Comment:
Setting the page weight determines the order the Operator content appears in the Set up navigation. Setting weight: 375 means this page will appear after helm and before linux setup pages. 
```suggestion
weight: 375
```